packages/icon/src/vaadin-icon.js

```diff
@@ -141,6 +143,18 @@ class Icon extends ThemableMixin(ElementMixin(ControllerMixin(ConditionalResizeM
         type: Object,
       },
 
+      /**
+       * The SVG source to be loaded as the icon. It can be:
+       * - an URL to a file containing the icon
+       * - an URL in the format "/path/to/file.svg#objectID", where the "objectID" refers to an ID attribute contained
+       *   inside the SVG referenced by the path. Note that the file needs to follow the same-origin policy.
+       * - a string in the format "data:image/svg+xml,<svg>...</svg>". You may need to use the "encodeURIComponent"
+       *   function for the SVG content passed
+       */
+      src: {
```

Comment:
Missing definition in the corresponding d.ts file

Reply:
Done.

Reply:
Also, add a [typings test](https://github.com/vaadin/web-components/pull/6394/files#diff-450e5e72f624894458732de388a315b4fd08e7cb4eee8a6e6a9cbd3ddcc5a5c7R12) for it